blob-csi-1.26.yaml

```diff
@@ -1,17 +1,27 @@
 package:
   name: blob-csi-1.26
   version: "1.26.5"
-  epoch: 52
+  epoch: 53
   description: Azure Blob Storage CSI driver
   copyright:
     - license: Apache-2.0
   dependencies:
     provides:
       - blob-csi=${{package.full-version}}
     runtime:
+      - bash
       - merged-lib
       - wolfi-baselayout
 
+vars:
+  # --- Manual update required ---
+  # This application depends on aznfs-mount project, which are contained
+  # in a different repository and each blob-csi release is using a specific
+  # version of aznfs-mount project so you need to manually update the
+  # aznfsVer variable below to match the version of aznfs-mount that
+  # blob-csi is using: https://github.com/kubernetes-sigs/blob-csi-driver/blob/v1.26.5/pkg/blobplugin/Dockerfile#L27
+  aznfsVer: "2.0.12"
```

Comment:
what do you think of getting the version here like this?

```bash
awk -F'=' '/aznfsVer/ {print $2}' ./pkg/blobplugin/Dockerfile
```

we get this version; export it and then do manual git clone. 

I think this way we will also reduce the chances of relying in manual updates. The machine will do the job and if it fails then we'll encounter build failures.

Reply:
We did this intentionally, as we wanted to avoid maintaining multiple versions of aznfs-mount, especially since it's not a version-streamed package. The main reason is that blob-csi uses different versions of aznfs-mount, so keeping the aznfs-mount version within the main package makes it much simpler to track and manage the relationship between the two, please tell me what do you think?